src/syncerd/monero_syncer.rs

```diff
@@ -71,24 +71,22 @@ pub struct Transaction {
     block_hash: Option<Vec<u8>>,
 }
 
+fn create_monero_rpc_client(rpc_url: String, proxy_url: Option<String>) -> monero_rpc::RpcClient {
```

Comment:
As we are in `monero_syncer` module we could reduce this fn name to `create_rpc_client`

Reply:
f72421341c872bf90b071a560e24abbc9872a62d